Model01-Firmware.ino

```diff
@@ -488,6 +508,40 @@ void setup() {
   // This avoids over-taxing devices that don't have a lot of power to share
   // with USB devices
   LEDOff.activate();
+
+  // To make the keymap editable without flashing new firmware, we store an
+  // overlay in EEPROM. For this to work, we need two things:
+
+  // We need to tell the firmware that keys should be looked up in EEPROM first,
+  // but transparent keys should fall back to the hard-coded keymap. With this
+  // arrangement, the firmware will use the built-in keymap by default, but still
+  // allow it to be changed by updating the overlay in EEPROM.
+  Layer.getKey = EEPROMKeymap.getKeyOverride;
+
+  // Second, we need to tell the plugin how many keymaps to store in EEPROM.
+  // We'll store three, the same amount we have built-in. It's a good idea to
```

Comment:
Why is it a good idea? I feel like defaulting to 6 may make users' lives better.